packages/components/src/local/atoms/game-title/index.tsx

```diff
@@ -7,20 +7,20 @@ import Props from './types/props'
 import styles from './styles.module.scss'
 
 /* Render component */
-export const GameTitle: React.FC<Props> = ({ initiated, title, name, onClick }: Props) => {
+export const GameTitle: React.FC<Props> = React.forwardRef(({ initiated, title, name, onClick }: Props, ref) => {
```

Comment:
@sanjayradadiya  - just to help my React knowledge, could you explain what `forwardRef` is doing here please?

Reply:
It will help to return a reference from `GameTitle` component , you can see I am trying to access the reference of span element of `GameTitle`.